README.md

```diff
@@ -1,5 +1,7 @@
 # OCSF Examples
 
+DISCLAIMER: This repository is by no means a 'gold standard' for OCSF mappings. It serves as a starting point for examples of how logs can be interpreted and mapped to OCSF. These examples are not vendor approved and should be used for informational purposes only.
```

Comment:
re. 
>These examples are not vendor approved

Some of them are vendor approved.
We should update this [readme](https://github.com/ocsf/examples/blob/main/mappings/README.md), perhaps in a tabular format and add a column to notify which are approved and which are community provided. Not saying it should be a part of this PR, but just a note.

Reply:
Alright. I can take this specific note out

Reply:
@floydtree That part was removed.